jobs/resource_job.go

```diff
@@ -208,7 +198,8 @@ type JobTaskSettings struct {
 	// ConditionTask is in private preview
 	ConditionTask *jobs.ConditionTask `json:"condition_task,omitempty" tf:"group:task_type"`
 
-	EmailNotifications     *EmailNotifications            `json:"email_notifications,omitempty" tf:"suppress_diff"`
+	EmailNotifications     *jobs.TaskEmailNotifications   `json:"email_notifications,omitempty" tf:"suppress_diff"`
+	WebhookNotifications   *WebhookNotifications          `json:"webhook_notifications,omitempty" tf:"suppress_diff"`
```

Comment:
Can we not use the SDK for Webhook Notifications? If you need the Sort() method, I was thinking something like
```go
type WebhookNotifications struct {
  *jobs.WebhookNotifications
}

func (w *WebhookNotifications) Sort() {
}
```
could do the trick

Reply:
I'm thinking if we ever need sorting - I think that we need to ask Jobs team to check if they are sorting the submitted values?

Reply:
@mgyucht we still need to sort, but I reimplemented it a bit differently - please look to updates